test/evergreen.yml

```diff
@@ -1521,6 +1543,8 @@ buildvariants:
     - name: make-check-asan-test
     - name: configure-combinations
     - name: checkpoint-filetypes-test
+    - name: compile-ppc
+    - name: test-unit-ppc
```

Comment:
These new tasks should be run on `ubuntu1604-power8-build` or `ubuntu1604-power8-test` distros.